src/util/error.js

```diff
@@ -7,3 +7,7 @@
 export default function getBadItemError(): Error {
```

Comment:
I would remove the default one now that this file has more, and replace its usage with { ... } style